cirq/experiments/fidelity_estimation.py

```diff
@@ -642,3 +646,195 @@ def per_cycle_depth(df):
         return pd.Series({'fidelity': fid_lsq})
 
     return df.reset_index().groupby('cycle_depth').apply(per_cycle_depth).reset_index()
+
+
+# mypy issue: https://github.com/python/mypy/issues/5374
+@dataclass(frozen=True)  # type: ignore
+class XEBPhasedFSimCalibrationOptions:
+    """Options for calibrating a PhasedFSim-like gate using XEB.
+
+    You may want to use more specific subclasses like `SqrtISwapXEBOptions`
+    which have sensible defaults.
+
+    Attributes:
+        parameterize_theta: Whether to characterize θ angle.
```

Comment:
Is there a reason to call this parametrize_? In the beginning I was using estimate_ but based on feedback from some people we settled down on characterize_

Reply:
changed to characterize